### PR TITLE
[DRAFT] Add xt-core v0.1.0 release notes

### PR DESCRIPTION
**Pull request by rshym**
This is initial revision of the document.

### CODE REVIEW

File: xt-core-RC-0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+```
+XT_DOMU_DTB_NAME = "%{XT_DOMU_DTB_NAME}"
+XT_DOMU_CONFIG_NAME = "%{XT_DOMU_CONFIG_NAME}"
+```
+
+If DomA is present
+```
+XT_DOMA_DTB_NAME = "%{XT_DOMA_DTB_NAME}"
+XT_DOMA_CONFIG_NAME = "%{XT_DOMA_CONFIG_NAME}"
+OVERRIDES:append = ":enable_android"
```
Comment (lorc) [inline xt-core-RC-0.1.0.md]: Should you define `XT_DOMA_TAG` as well? Those tags were used as a easy way to tell build system which domains are being built.

File: xt-core-RC-0.1.0.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+```
+
+### Variables required to be set for DomD
+```
+MACHINE = "%{DOMD_MACHINE}"
+XT_DOM_NAME = "domd"
+XT_OP_TEE_FLAVOUR = "%{XT_OP_TEE_FLAVOUR}"
+XT_DEVICE_TREES = "%{XT_DOMD_DTB_NAME} %{XT_DOMA_DTB_NAME} %{XT_XEN_DTB_NAME}"
+XT_GUEST_INSTALL = "%{XT_GENERIC_DOMU_TAG} %{XT_DOMA_TAG} %{XT_DOMZ_TAG}"
+XT_MULTIMEDIA_EVA_DIR = "%{XT_MULTIMEDIA_EVA_DIR}"
```
Comment (lorc) [inline xt-core-RC-0.1.0.md]: That's very Gen3 specific, I guess it needs some comment about this

File: xt-core-RC-0.1.0.md (hunk 1 above)
Comment (rshym) [reply xt-core-RC-0.1.0.md]: Thanks, you are right. I have added XT_DOMA_TAG, XT_DOMU_TAG and XT_DOMZ_TAG as well.

File: xt-core-RC-0.1.0.md (hunk 2 above)
Comment (rshym) [reply xt-core-RC-0.1.0.md]: Thanks. I have added info for multimedia and graphic packages, that they are R-Car Gen3 specific proprietary packages, and are not a part of the release.

**Comment by rshym** [comment]
This PR is just POC and is not intended to be merged.

**Pull request closed without merging**